app/models/issue.rb

```diff
@@ -59,4 +60,25 @@ def can_read_when_unpublished?(user)
   def can_read?(user)
     published? || can_read_when_unpublished?(user)
   end
+
+  def fb_post
+    { message: fb_message, link: fb_link, name: title.to_s,
+      picture: fb_picture }
+  end
+
+  def fb_message
+    fb_location = location.blank? ? '' : "Адреса: #{location} \n \n"
+    fb_description = description.blank? ? '' : "Опис: #{description} \n \n"
+    tags = category.tags.blank? ? '' : category.tags.to_s
+    [fb_location, fb_description, tags].reject(&:blank?).join(' ')
+  end
+
+  def fb_link
+    issue_url(self, host: Rails.application.config.host)
+  end
+
+  def fb_picture
+    picture = attachment.file.nil? ? '/uploads/default-image.jpg' : attachment
```

Comment:
we use it in next line to create link to the image:

"#{ENV['IMAGE_HOSTING_URL']}#{**picture**}"